.github/workflows/molecule.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Molecule
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scenario:
+          - centos-6
+          - centos-7
+          - centos-8
+          - debian-8
+          - debian-9
+          - debian-10
+          - fedora-26
+          - fedora-27
+          - fedora-28
+          - fedora-29
+          - fedora-30
+          - fedora-31
+          - oraclelinux-6
+          - oraclelinux-7
+          # - oraclelinux-8
```

Comment:
disabled `oraclelinux-8` due to failure with some error regarding systemd
working on fix in docker container